solutions/community-deploy/cloud/AWS/README.md

```diff
@@ -48,66 +43,96 @@ os_version          = "CentOS 7.9.2009 x86_64"
 ```
 terraform validate && terraform apply -var-file user.tfvars --auto-approve
 ```
+- Execute following commands which will perform on all the nodes in the cluster,
+  - Setup network and storage devices for CORTX.
+  - Prompt for generating the `root` user password
+  - `/home/centos/setup.sh` will reboot all the nodes once executed
 
-## Network and Storage Configuration.
+**Note:**
+*The root password is required as a part of CORTX deployment.*
+```
+PUBLIC_IP=`terraform show -json terraform.tfstate | jq .values.outputs.aws_instance_public_ip_addr.value 2>&1 | tee ip.txt  | tr -d '",[]' | sed '/^$/d'`
+for ip in $PUBLIC_IP;do
+   ssh -i cortx.pem -o 'StrictHostKeyChecking=no' centos@$ip 'echo "Enter new password for root user" && sudo passwd root && sudo bash /home/centos/setup.sh'
+   ssh -i cortx.pem -o 'StrictHostKeyChecking=no' centos@$ip 'echo "Enter new password for root user" && sudo passwd root && sudo bash /home/centos/setup.sh'
+   ssh -i cortx.pem -o 'StrictHostKeyChecking=no' centos@$ip 'echo "Enter new password for root user" && sudo passwd root && sudo bash /home/centos/setup.sh'
+done
+```
+- AWS instances are ready for CORTX Build and deployment now. Connect to EC2 nodes over SSH and validate that all three network cards has IP address assigned.
 
-- Execute `/home/centos/setup.sh` to setup Network and Storage devices for CORTX. Script will reboot instance on completion. 
+### CORTX Build
 
+- We will use [cortx-build](https://github.com/Seagate/cortx/pkgs/container/cortx-build) docker image to compile entire CORTX stack.  
+- Become the root user by running the following command,
 ```
-ssh -i cortx.pem -o 'StrictHostKeyChecking=no' centos@"<AWS instance public-ip>" sudo bash /home/centos/setup.sh
+sudo su -
 ```
-- AWS instance is ready for CORTX Build and deployment now. Connect to instance over SSH and validate that all three network cards has IP address assigned.
-   
-- Generate `root` user password.  
-*The root password is required as a part of CORTX deployment.*
-   
+- Login to all the nodes over SSH from local host using public IP address and clone the cortx-re repository
+```
+ssh -i cortx.pem -o 'StrictHostKeyChecking=no' centos@"<AWS instance public-ip-primarynode>" 'git clone https://github.com/Seagate/cortx-re'
+ssh -i cortx.pem -o 'StrictHostKeyChecking=no' centos@"<AWS instance private-ip-workernode1>" 'git clone https://github.com/Seagate/cortx-re'
+ssh -i cortx.pem -o 'StrictHostKeyChecking=no' centos@"<AWS instance private-ip-workernode2>" 'git clone https://github.com/Seagate/cortx-re'
+```
+- Execute `build-cortx.sh` from primary node which will generate CORTX container images from `main` of CORTX components
+```
+cd $PWD/cortx-re/solutions/community-deploy
+ssh -i cortx.pem -o 'StrictHostKeyChecking=no' centos@"<AWS instance public-ip-primarynode>" sudo time build-cortx.sh
+```
+- Save and compress the cortx build images
```

Comment:
I feel setting up a local registry would be easier than executing so many commands. Maybe we can take it as improvement later.